### PR TITLE
Fixed Modern email theme for RTL languages

### LINKED ISSUE #26177: [BO] Modern email theme isn't optimized for RTL languages
#### Describe the bug

Email templates that is generated base on Modern theme, isn't configured for RTL languages.

#### Expected behavior

Text align in email templates for RTL languages must be from right to left.

#### Steps to Reproduce

1. BO language must be RTL
2. Go to BO > Themes > Email theme
2. At "Generate emails" section, select Modern theme
3. Select a RTL language
4. Generate emails
4. At bottom of the current page ("Email themes" section) check email templates for Modern theme

**Screenshots**

Generated email template:

Expected email template:

#### Additional information

* PrestaShop version: 1.7.8.0
* PHP version: 7.3

### PR DESCRIPTION
**Pull request by mparvazi**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixes text align and text direction in Modern email template for RTL languages.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #26177
| How to test?      | Email template must be regenerate base on Modern theme and a RTL language.
| Possible impacts? | -

**Screenshots**

Generated email template:

Expected email template:

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26340)

### CODE REVIEW

File: mails/themes/modern/components/layout.html.twig [hunk 1]
```diff
@@ -55,6 +55,24 @@
         </style>
         <![endif]-->
   <style type="text/css">
+  {% if languageIsRTL|default(false) %}
```
Comment (Progi1984) [inline mails/themes/modern/components/layout.html.twig:58]: @mparvazi Could you add the `style` tag in the `if` condition ?

File: mails/themes/modern/components/order_layout.html.twig [hunk 2]
```diff
@@ -55,6 +55,24 @@
         </style>
         <![endif]-->
   <style type="text/css">
+  {% if languageIsRTL|default(false) %}
```
Comment (Progi1984) [inline mails/themes/modern/components/order_layout.html.twig:58]: @mparvazi Could you add the `style` tag in the `if` condition ?

File: mails/themes/modern/components/layout.html.twig (hunk 1 above)
Comment (Progi1984) [reply mails/themes/modern/components/layout.html.twig:58]: Better remove the current style tag and move it to the tag `style` under this current one

File: mails/themes/modern/components/order_layout.html.twig (hunk 2 above)
Comment (Progi1984) [reply mails/themes/modern/components/order_layout.html.twig:58]: Better remove the current style tag and move it to the tag `style` under this current one

**Review by Progi1984 — COMMENTED** [review]
@mparvazi Thank you for your contribution. I have brought you the same comment in two files. Could you check it ?

File: mails/themes/modern/components/layout.html.twig [hunk 3]
```diff
@@ -54,8 +54,24 @@
           .mj-outlook-group-fix { width:100% !important; }
         </style>
         <![endif]-->
-  <style type="text/css">
-  </style>
+  {% if languageIsRTL|default(false) %}<style type="text/css">
+    div {
```
Comment (Progi1984) [inline mails/themes/modern/components/layout.html.twig]:
```suggestion
  <style type="text/css">
  {% if languageIsRTL|default(false) %}
    div {
```

File: mails/themes/modern/components/layout.html.twig [hunk 4]
```diff
@@ -54,8 +54,24 @@
...
+    
+    .mj-column-px-25 {
+      float: right;
+    }
+    
+    .footer div {
+      text-align: center !important;
+    }
+  </style>{% endif %}
   <style type="text/css">
```
Comment (Progi1984) [inline mails/themes/modern/components/layout.html.twig]:
```suggestion
  {% endif %}
```

File: mails/themes/modern/components/order_layout.html.twig [hunk 5]
```diff
@@ -54,8 +54,24 @@
           .mj-outlook-group-fix { width:100% !important; }
         </style>
         <![endif]-->
-  <style type="text/css">
-  </style>
+  {% if languageIsRTL|default(false) %}<style type="text/css">
+    div {
```
Comment (Progi1984) [inline mails/themes/modern/components/order_layout.html.twig]:
```suggestion
  <style type="text/css">
  {% if languageIsRTL|default(false) %}
    div {
```

File: mails/themes/modern/components/order_layout.html.twig [hunk 6]
```diff
@@ -54,8 +54,24 @@
...
+    
+    .mj-column-px-25 {
+      float: right;
+    }
+    
+    .footer div {
+      text-align: center !important;
+    }
+  </style>{% endif %}
   <style type="text/css">
```
Comment (Progi1984) [inline mails/themes/modern/components/order_layout.html.twig]:
```suggestion
  {% endif %}
```

**Review by Progi1984 — COMMENTED** [review]
A last comment

**Comment by mparvazi** [comment]
@Progi1984 Thanks!
I think it's good now.

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — CHANGES_REQUESTED** [review]
Changes on the email theme should be made in https://github.com/PrestaShop/mjml-theme-converter, then using mjml-theme-converter you can generate the .twig files for the core.

**Comment by mparvazi** [comment]
@atomiix A new PR at https://github.com/PrestaShop/mjml-theme-converter ?

**Comment by atomiix** [comment]
@mparvazi yes, please :)

**Pull request closed without merging**